src/services-managing.rst

```diff
@@ -61,49 +61,19 @@ Before you are able to synchronize data or application files to a device, you wi
 
       If you are using SSL, be sure to specify :code:`https://...`
 
-    .. _services-login-with-qr:
```

Comment:
Has the QR code feature been removed? Just want to check on that before deleting this text - I thought I saw a screenshot of the QR code still.

Reply:
No, but there's a section on the services using page so I just referenced that.